002-es-modules.md

```diff
@@ -0,0 +1,175 @@
+| Title  | ESM Interoperability |
+|--------|-----------------------------|
+| Author | @bmeck                      |
+| Status | DRAFT                       |
+| Date   | 2016-08-26                  |
+
+**NOTE:** `DRAFT` status does not mean ESM will be implemented in Node
+core. Instead that this is the standard, should Node core decide to implement
+ESM. At which time this draft would be moved to `ACCEPTED`.
+
+---
+
+Abbreviations:
+* `ESM` - Ecma262 Modules (ES Modules)
+* `NCJS` - Node Modules (a CommonJS variant)
+* `NM2` - Node v2.0 Modules (Node Modules using the Module parse goal)
+* `RMR` - Reflective Module Record (`ModuleRecord` type)
+
+The intent of this standard is to:
+
+* implement interoperability for the Module Parse Goal and Node's existing
+module system
+
+## 1. Purpose
+
+1. Allow Node.js modules to use "Module Mode" without hitting the Reflective
+Module Record.
+  - The Reflective Module Record is described below as the `ModuleRecord` type.
+2. Allow Node.js modules to use "Module Mode" without using Async Module Resolution.
```

Comment:
Just skipped through the spec and tried to find other things that depend on the goal symbol (other than `await`) and came up empty. :( Is there a list somewhere?